packages/@storefront/flux-capacitor/src/core/sagas/recommendations.ts

```diff
@@ -132,7 +128,8 @@ export namespace RecommendationsTasks {
           flux.actions.receivePastPurchaseCurrentRecordCount(results.totalRecordCount),
           flux.actions.receivePastPurchaseProducts(SearchAdapter.augmentProducts(results)),
           flux.actions.receivePastPurchaseTemplate(SearchAdapter.extractTemplate(results.template)),
-          flux.actions.receivePastPurchaseRefinements(SearchAdapter.combineNavigations(results)),
+          flux.actions.receivePastPurchaseRefinements(
+            SearchAdapter.pruneRefinements(SearchAdapter.combineNavigations(results), state)),
```

Comment:
If you want to use `pruneRefinements` here (which you probably _should_) you need to either add a `maxRefinements` configuration for PP, or pass the store section to `pruneRefinements` and don't use a max for PP. Otherwise, `pruneRefinements` will apply the `search.maxRefinements` configuration, which will probably be confusing (and a bug) to users

Reply:
Updated `pruneRefinements` to receive a `storeSection` param. Param used to extract the correct `maxRefinements` value from the config.